commands/secret_list.go

```diff
@@ -40,6 +40,7 @@ func preRunSecretListCmd(cmd *cobra.Command, args []string) error {
 func runSecretList(cmd *cobra.Command, args []string) error {
 	var gatewayAddress string
 	gatewayAddress = getGatewayURL(gateway, defaultGateway, "", os.Getenv(openFaaSURLEnvironment))
+	fmt.Print(checkTLSInsecure(gatewayAddress, tlsInsecure), "\n")
```

Comment:
use `fmt.Println()`
```suggestion
	fmt.Println(checkTLSInsecure(gatewayAddress, tlsInsecure))
```

Reply:
oh the `Println` was a mistake I had something different and forgot to revert back to `println` I will fix

Reply:
👍